drivers/tty/serdev/pibridge_trace.h

```diff
@@ -41,16 +51,40 @@ DECLARE_EVENT_CLASS(pibridge_gate_header_class,
 	)
 );
 
+/*
+ * pibridge_send_gate_header
+ *
+ * Info: the header of a gateway packet to be sent.
+ * Time: before the gate packet header is sent on the wire.
+ */
 DEFINE_EVENT(pibridge_gate_header_class, pibridge_send_gate_header,
 	TP_PROTO(struct pibridge_pkthdr_gate *hdr),
 	TP_ARGS(hdr)
 );
 
+/*
+ * pibridge_receive_gate_header
+ *
+ * Info: the header of a received gateway packet.
+ * Time: after the gate packet header was received from the wire.
+ */
 DEFINE_EVENT(pibridge_gate_header_class, pibridge_receive_gate_header,
 	TP_PROTO(struct pibridge_pkthdr_gate *hdr),
 	TP_ARGS(hdr)
 );
 
+/*
+ * pibridge_io_header_class
+ *
+ * Print the elements of an IO packet header.
+ *
+ * addr: destination address. Specifies to which RevPi device the packet is sent
+ *       to.
+ * type: message type.
+ * rsp: specifies if this message is a request or a response.
```

Comment:
It says "message" here and in the preceding line, but "packet" everywhere else. Maybe change this to "packet" as well for uniformity?